src/packages/upgrading.md

```diff
@@ -0,0 +1,46 @@
+# Upgrading packages
+
+It is recommended to upgrade packages in the following situations:
+
+* Before building a newer CheriBSD release;
+
+  The release might require an updated toolchain (e.g., due to ABI changes in
+  the release that must by supported by the toolchain).
+
+* After updating your host to a newer CheriBSD release;
+
+  A package repository for the release is compiled against an ABI version
+  supported by the newer release that might have changed since its prior
+  release.
```

Comment:
```suggestion
  Package repositories are compiled for the ABI version
  used by the corresponding release, which might be newer than the previous
  release.
```

Reply:
I've rephrased that a bit not to repeat "newer". Also, it's probably better no to say that one ABI is newer than another because it's not something documented on its own. A user does not have a way to keep track of the history of ABI versions to compare them. Hence, I'd prefer not to confuse the user and just say they can differ.